guides/CHANGELOG.md

```diff
@@ -7,4 +7,10 @@
 
     *Petrik de Heus*
 
+*   The guide _Active Record Multiple Databases_ has been updated to explain that
+    when you switch to a multi database setup, the DATABASE_URL environment
+    variable is no longer automatically used for the primary database.
+
+    *Hampton Lintorn-Catlin*
+
```

Comment:
```suggestion
```


I forgot the guides had a changelog but I don't think it is necessary for this change. Maybe for big changes like design or removing pages.